global.json

```diff
@@ -1,3 +1,6 @@
 {
-  "projects": [ "src" ]
+  "projects": [ "src" ],
+    "sdk": {
+    "version": "1.0.0-preview2-003121"
```

Comment:
I couldn't even compile without it when I first started looking at ImageSharp... I'll see if it's still needed.